custom/icds_reports/dashboard_utils.py

```diff
@@ -13,6 +13,13 @@ def get_dashboard_template_context(domain, couch_user):
     context['all_user_location_id'] = list(couch_user.get_sql_locations(
         domain
     ).location_ids())
+    try:
+        if couch_user.is_commcare_user():
+            context['user_location_type'] = couch_user.sql_location.location_type_name
+        else:
+            context['user_location_type'] = couch_user.get_sql_location(domain).location_type_name
+    except AttributeError:
```

Comment:
Why is this exception?

Reply:
there are chances that some of the webusers does not have a location. while trying to retreive location type for those, we will get this exception